sdk/core/System.ServiceModel.Rest/src/Temporary/Utf8JsonRequestBody.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable enable
+
+using System.IO;
+using System.ServiceModel.Rest.Core;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace System.ServiceModel.Rest.Shared.Core.Serialization
+{
+    public class Utf8JsonRequestBody : RequestBody
+    {
+        private readonly MemoryStream _stream;
+        private readonly RequestBody _content;
+
+        public Utf8JsonWriter JsonWriter { get; }
+
+        public Utf8JsonRequestBody()
+        {
+            _stream = new MemoryStream();
+            _content = CreateFromStream(_stream);
```

Comment:
This method call will go back to Create when we can clean up the shared source story.